examples/multiple-accounts/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2024 Ori Pomerantz <qbzzt1@gmail.com>
```

Comment:
we don't have a licence file in the other example projects bc they inherit the monorepo root's license (MIT), don't think we need a separate one here